tensorflow/compiler/xla/service/gpu/multi_output_fusion.cc

```diff
@@ -155,9 +152,7 @@ bool GpuMultiOutputFusion::DoProducerConsumerMultiOutputFusion() {
         VLOG(3) << producer->name() << " is a constant.";
         continue;
       }
-      const bool is_loop_fusion =
-          producer->opcode() == HloOpcode::kFusion &&
-          producer->fusion_kind() == HloInstruction::FusionKind::kLoop;
+      const bool is_loop_fusion = producer->IsLoopFusion();
```

Comment:
Nit: Please inline this.

Reply:
Would you like me to turn "IsLoopFusion()" into an inline function? Or replace is_loop_fusion with calls to IsLoopFusion()?

Reply:
My idea was to replace line 156 with
`if (!producer->IsElementwise() &&  !producer->IsLoopFusion())`